sdcm/mgmt/common.py

```diff
@@ -22,6 +22,7 @@ def get_distro_name(distro_object):
         Distro.AMAZON2023: "centos8",
         Distro.CENTOS7: "centos7",
         Distro.CENTOS8: "centos8",
+        Distro.CENTOS9: "centos8",
```

Comment:
shouldn't it be centos9?

Reply:
it's the same, i.e. point to the same RPMs repo url.